app/api/server/v1/misc.js

```diff
@@ -202,3 +203,9 @@ API.v1.addRoute('directory', { authRequired: true }, {
 		});
 	},
 });
+
+API.v1.addRoute('stdout.queue', { authRequired: true }, {
+	get() {
+		return API.v1.success({ queue: StdOut.queue });
```

Comment:
need to check [view-log](https://github.com/RocketChat/Rocket.Chat/pull/16004/files#diff-fef08fa7f2b42bae3070c24300a6f941R74) permissions